content/en/docs/concepts/services-networking/service.md

```diff
@@ -203,10 +203,13 @@ described in detail in [EndpointSlices](/docs/concepts/services-networking/endpo
 
 ### Application protocol
 
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```

Comment:
You could leave this line in and mark it stable for v1.20 so people know to look at older docs if their cluster isn't bang up to date.

Reply:
Good call, added this back in, thanks!